targets/unity-v2/source/Entity/ScreenTimeTracker.cs

```diff
@@ -173,7 +190,7 @@ private void EventSentErrorCallback(PlayFabError response)
             Debug.LogWarning("Failed to send session data. Error: " + response.GenerateErrorReport());
         }
 
-#region Unused MonoBehaviour compatibility  methods
+        #region Unused MonoBehaviour compatibility  methods
```

Comment:
Please use the Visual Studio "Ctrl+k+d" standard spacing.

Reply:
Used VS standard spacing, no difference. I'd assume "standard" is not so standard and we need to sync our settings.